### DISCUSSION TITLE
Dealing with multiple poster

### DISCUSSION THREAD

**Discussion by nicolasrata** [meisnate12/Plex-Meta-Manager#923] [Q&A] [answered]
Hi,
How do you deal with the multiple poster in the metadata folder.

For some movies I have 4 or more file in the Uploads folder
![Capture d’écran 2022-06-10 à 19 24 06](https://user-images.githubusercontent.com/20968235/173119166-dcf72a99-4ca5-4cef-af87-07fe4b8b61f9.png)

![Capture d’écran 2022-06-10 à 19 27 21](https://user-images.githubusercontent.com/20968235/173119662-728c5291-77c2-40d6-84a5-6410429a4dc1.png)

Any idea how to remove the unnecessary files?

Thanks

**Comment by chazlarson** [comment]
That's an open question that is being worked on.  There is discussion on the Discord around it.  There are scripts that claim to address this with variable levels of success.  Plex Dancing the item will clear the excess posters, as a brute force method.

**Reply by nicolasrata** [reply]
Thanks for the very quick answer

**Comment by nicolasrata** [comment, marked as answer]
Here is the solution

[https://github.com/Casvt/Plex-scripts/blob/main/various/remove_unused_artwork.py](url)
